.github/workflows/elixir.yml

```diff
@@ -161,14 +161,14 @@ jobs:
       # and running the workflow steps.
       matrix:
         include:
-          - elixir: "1.15.3-otp-26"
-            otp: "26.0.2"
-          - elixir: "1.15.3-otp-25"
-            otp: "25.3"
-          - elixir: "1.14.5-otp-25"
-            otp: "25.3"
-          - elixir: "1.13.4-otp-25"
-            otp: "25.3"
+          - elixir: "1.15"
```

Comment:
this has the unfortunate side effect of being on the latest patchlevel version when it drops, and has bit us before. I suggest using `1.15.6` for now.